.github/workflows/draft.yml

```
@@ -0,0 +1,18 @@
+name: Create Preview Draft from PR
+on:
+  pull_request:
+    branches: ['main']
+permissions:
+  contents: read
+  pull-requests: write
+jobs:
+  draft:
+    uses: curvenote/actions/.github/workflows/draft.yml@v1
+    with:
+      id-pattern-regex: '^physiome-(?:[a-zA-Z0-9-_]{3,40})$'
```

We could make this validation more specific too. I think all the current articles conform to:
```suggestion
      id-pattern-regex: '^physiome-S(?:[0-9]{6})$'
```